oci/model.py

```diff
@@ -43,8 +43,24 @@ class OciRegistryType(enum.Enum):
     GCR = 'gcr'
     DOCKERHUB = 'dockerhub'
     ARTIFACTORY = 'artifactory'
+    GHCR = 'ghcr'
+    ALIBABA = 'alibaba'
```

Comment:
nit: keep abbreviations consistent and use `ACR` or `ACCR` for `Alibaba Cloud Container Registry`

Reply:
isn't it rather AliYun / AliCloud?

Reply:
also, I would prefer not to introduce abbreviations, such as ACR and rather go through the lengths of spending a few more characters for the sake of being more descriptive ("well-known" abbreviations are okay)

Reply:
was refering to this document:
https://www.alibabacloud.com/help/en/acr/product-overview/what-is-container-registry